app/views/mailers/notify_mailer/new_badge_email.text.erb

```diff
@@ -6,8 +6,10 @@ Congratulations, <%= @user.name %>! You got the <%= @badge.title %> badge! Be su
 
 <%= user_url(@user) %>
 
-You also get 5 new credits to use for community listings if you have anything you'd like to promote: <%= listings_url %>
+<% unless @badge.credits_awarded.zero? %>
```

Comment:
**suggestion (non-blocking):** Same suggestion as above:

```suggestion
<% if @badge.credits_awarded.positive? %>
```

Reply:
I'm not fan of the `positive?` in such cases because I start thinking that the number could possibly be negative when I read such code 😅 In this case I mean `nonzero` but it's also negative :)
I won't mind this change in the future but will keep it as is in the current pr.